src/main/kotlin/study/StringAddCalculator.kt

```diff
@@ -0,0 +1,25 @@
+package study
+
+class StringAddCalculator {
```

Comment:
파싱하는 것과 계산하는 역할을 객체로 나눠봐도 좋겠어요!

또, 가변 상태를 담는 멤버가 없으니, object키워드를 활용해보셔도 좋겠어요 :)
```suggestion
object StringAddCalculator {
```